### PR TITLE
Board

### PR DESCRIPTION
**Pull request by nik-1207**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 build
 node_modules
 yarn.lock
-public
\ No newline at end of file
+public
```
Comment (AbhayMaheshwari123) [inline .prettierignore:4]: unnecessary change

File: tsconfig.json [hunk 2]
```diff
@@ -28,4 +28,4 @@
     "coverage",
     "build"
   ] // *** The files to not type check ***
-}
\ No newline at end of file
+}
```
Comment (AbhayMaheshwari123) [inline tsconfig.json:31]: unnecessary chage

File: src/components/Board.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Grid, Paper } from "@mui/material";
+
+const formatNumber: (num: number) => string = (num: number) =>
+  num === 0 ? "_" : num < 10 ? `0${num}` : `${num}`;
+
+const Board: React.FC = () => {
+  const arr: Array<number> = React.useMemo(
+    () => Array.from({ length: 90 }, (_) => 0),
```
Comment (AbhayMaheshwari123) [inline src/components/Board.tsx]: value given to length should be in util file

File: src/components/Board.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Grid, Paper } from "@mui/material";
+
+const formatNumber: (num: number) => string = (num: number) =>
```
Comment (AbhayMaheshwari123) [inline src/components/Board.tsx]: should be in different file i.e., helper file

**Review by AbhayMaheshwari123 — COMMENTED** [review]
Some changes to do.

File: .prettierignore (hunk 1 above)
Comment (nik-1207) [reply .prettierignore:4]: I only added line 5 and 6 but showing line 4  is also changed

Comment (nik-1207) [reply .prettierignore:4]: I only added line 5 and 6 showing line 4 also changed

File: tsconfig.json (hunk 2 above)
Comment (nik-1207) [reply tsconfig.json:31]: prettier --fix did this

File: src/components/Board.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import React from "react";
+
+const arr = Array.from({ length: 90 }, (_, i) => i + 1);
+
+const numberToBeDisplayed: (num: number) => string = (num: number) =>
+  num < 10 ? `0${num}` : `${num}`;
+
+const Board: React.FC = () => {
+  return (
+    <Grid
```
Comment (Parm-S) [inline src/components/Board.tsx:12]: wrap the grid component around paper

File: src/App.tsx [hunk 6]
```diff
@@ -1,11 +1,16 @@
 import React from "react";
 import "./App.css";
```
Comment (Parm-S) [inline src/App.tsx:2]: why app.css

Comment (nik-1207) [reply src/App.tsx:2]: to center the generated numbers

File: src/components/Board.tsx (hunk 5 above)
Comment (nik-1207) [reply src/components/Board.tsx:12]: because if item is not of type paper there is distortion in ui like this

image
without paper it works like this

Comment (nik-1207) [reply src/components/Board.tsx:12]: you cant add padding between grid items cells

File: src/components/Board.tsx (hunk 3 above)
Comment (nik-1207) [reply src/components/Board.tsx]: completed

File: src/components/Board.tsx (hunk 4 above)
Comment (nik-1207) [reply src/components/Board.tsx]: completed

File: src/App.tsx (hunk 6 above)
Comment (nik-1207) [reply src/App.tsx:2]: should came from theme itself
